autopush/websocket.py

```diff
@@ -74,7 +74,10 @@ def f():
 
     @property
     def base_tags(self):
-        return self._base_tags if self._base_tags else None
+        try:
+            return self._base_tags
+        except AttributeError:
+            return None
```

Comment:
Not sure what case can cause this, also, the effect is not the same as before. The other return statement causes the return value to be None if self._base_tags is [], this is because there was a bug with the datadog library requiring it to be None instead of [].